test/testsets.json

```diff
@@ -41,6 +41,9 @@
     { "name": "test_fs_open_read_sync_3.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
     { "name": "test_gpio_input.js", "skip": ["all"], "reason": "needs hardware" },
     { "name": "test_gpio_output.js", "skip": ["all"], "reason": "need user input"},
+    { "name": "test_https_get.js", "timeout": 40 },
+    { "name": "test_https_request_response.js", "timeout": 40 },
+    { "name": "test_https_timeout.js", "timeout": 40 },
```

Comment:
Since the HTTPS module is in the "exclude" of all platforms except Tizen, and Tizen is not a platform offered by Travis, these test cases will never actually run for any platform. 
These test cases are here just for reference, so they can be used to test manually if needed as shown in [https://github.com/Samsung/iotjs/blob/master/docs/build/Build-for-ARTIK.md#testing](https://github.com/Samsung/iotjs/blob/master/docs/build/Build-for-ARTIK.md#testing)
Hence there is no need for a `"skip": ["all"]`, though I could add that in the JSON if you say so.